### PR TITLE
references: refine MySQL Replication Filtering Rules in DM Black & white table lists

### PR DESCRIPTION
**Pull request by csuzhangxc**
### What is changed, added or deleted? 

refine docs of DM Black & white table lists.

### What is the related PR or file link(s)? 

- add some explanation about Black & white table lists
- add some link to MySQL Replication Filtering Rules

### Which version does your change affect? 

dev/v3.0/v2.1

### CODE REVIEW

**Comment by csuzhangxc** [comment]
@WangXiangUSTC @lilin90 PTAL

File: dev/reference/tools/data-migration/features/overview.md [hunk 1]
```diff
@@ -127,14 +127,24 @@ black-white-list:
...
 
-- `do-dbs` 要同步的库的白名单
-- `ignore-dbs` 要同步的库的黑名单
-- `do-tables` 要同步的表的白名单
-- `ignore-tables` 要同步的表的黑名单
+- `do-dbs` 要同步的库的白名单，类似于 MySQL 中的 [`replicate-do-db`](https://dev.mysql.com/doc/refman/5.7/en/replication-options-slave.html#option_mysqld_replicate-do-db)
+- `ignore-dbs` 要同步的库的黑名单，类似于 MySQL 中的 [`replicate-ignore-db`](https://dev.mysql.com/doc/refman/5.7/en/replication-options-slave.html#option_mysqld_replicate-ignore-db)
+- `do-tables` 要同步的表的白名单，类似于 MySQL 中的 [`replicate-do-table`](https://dev.mysql.com/doc/refman/5.7/en/replication-options-slave.html#option_mysqld_replicate-do-table)
+- `ignore-tables` 要同步的表的黑名单，类似于 MySQL 中的 [`replicate-ignore-table`](https://dev.mysql.com/doc/refman/5.7/en/replication-options-slave.html#option_mysqld_replicate-ignore-table)
 - 上面黑白名单中 以 `~` 字符开头名称为[正则表达式](https://golang.org/pkg/regexp/syntax/#hdr-syntax)。
```
Comment (WangXiangUSTC) [inline dev/reference/tools/data-migration/features/overview.md]: 感觉这句话也不太容易理解，能否改一下，例如这样：
以上配置均支持使用正则表达式匹配库名、表名，需要以 `~` 字符开头。

File: dev/reference/tools/data-migration/features/overview.md [hunk 2]
```diff
@@ -127,14 +127,24 @@ black-white-list:
...
 
+`do-dbs` 与 `ignore-dbs` 对应的过滤规则与 MySQL 中的 [Evaluation of Database-Level Replication and Binary Logging Options](https://dev.mysql.com/doc/refman/5.7/en/replication-rules-db-options.html) 类似，`do-tables` 与 `ignore-tables` 对应的过滤规则与 MySQL 中的 [Evaluation of Table-Level Replication Options](https://dev.mysql.com/doc/refman/5.7/en/replication-rules-table-options.html) 类似。
+
+> **注意：**
+>
+> DM 中黑白名单过滤规则与 MySQL 中相应规则存在以下区别：
+>
+> - MySQL 中存在 [`replicate-wild-do-table`](https://dev.mysql.com/doc/refman/5.7/en/replication-options-slave.html#option_mysqld_replicate-wild-do-table) 与 [`replicate-wild-ignore-table`](https://dev.mysql.com/doc/refman/5.7/en/replication-options-slave.html#option_mysqld_replicate-wild-ignore-table) 用于支持通配符，DM 中各配置参数直接支持以 `~` 字符开头的正则表达式。
+> - DM 当前只支持 `ROW` 格式的 binlog，不支持 `STATEMENT`/`MIXED` 格式的 binlog，因此应与 MySQL 中 `ROW` 格式下的规则对应。
+> - 对于 DDL，MySQL 仅依据默认的 database 名称（`USE` 语句显式指定的 database）进行判断，而 DM 优先依据 DDL 中的 database 名称部分进行判断、并当 DDL 中不包含 database 名称时再依据 `USE` 部分进行判断。假设需要判断的 SQL 为 `USE test_db_2; CREATE TABLE test_db_1.test_table (c1 INT PRIMARY KEY)`，且 MySQL 配置了 `replicate-do-db=test_db_1`、DM 配置了 `do-dbs: ["test_db_1"]`，则对于 MySQL 该规则不会生效，而对于 DM 该规则会生效。
```
Comment (WangXiangUSTC) [inline dev/reference/tools/data-migration/features/overview.md]:
```suggestion
> - 对于 DDL，MySQL 仅依据默认的 database 名称（`USE` 语句显式指定的 database）进行判断，而 DM 优先依据 DDL 中的 database 名称部分进行判断，并当 DDL 中不包含 database 名称时再依据 `USE` 部分进行判断。假设需要判断的 SQL 为 `USE test_db_2; CREATE TABLE test_db_1.test_table (c1 INT PRIMARY KEY)`，且 MySQL 配置了 `replicate-do-db=test_db_1`、DM 配置了 `do-dbs: ["test_db_1"]`，则对于 MySQL 该规则不会生效，而对于 DM 该规则会生效。
```

**Comment by WangXiangUSTC** [comment]
rest LGTM

File: dev/reference/tools/data-migration/features/overview.md (hunk 1 above)
Comment (csuzhangxc) [reply dev/reference/tools/data-migration/features/overview.md]: 在 https://github.com/pingcap/docs-cn/pull/1895/commits/b9ad8a86b37eebd025e180048dc1df74f597abed 中改了一下

File: dev/reference/tools/data-migration/features/overview.md (hunk 2 above)
Comment (csuzhangxc) [reply dev/reference/tools/data-migration/features/overview.md]: done in https://github.com/pingcap/docs-cn/pull/1895/commits/b9ad8a86b37eebd025e180048dc1df74f597abed.

**Comment by WangXiangUSTC** [comment]
LGTM

**Pull request merged**